src/ashley/views.py

```diff
@@ -64,10 +64,18 @@ def _do_on_login(self, lti_request: LTI) -> HttpResponse:
                 lti_contexts__id=context.id,
             )
         except Forum.DoesNotExist:
+
+            # Check if this uuid exists for another context.id to reuse its name as a
+            # default value (enables copy and paste)
+            forum_same_uuid = Forum.objects.filter(
+                lti_id=forum_uuid, type=Forum.FORUM_POST
```

Comment:
aren't we mixing uuid and lti_id? In line 48 we set `lti_id=context_id` which seems wrong? There is a `lti_id` field in the request that doesn't change in OpenEdX accross course copies.

Reply:
you can add `only("name")` since that's the only field you're interested in here.

Reply:
The field lti_id in the forum table is the uuid of the forum ex 916180a7-0b43-5c08-b3c8-c738826880bb, the field lti_id in the LTIContext model is the id of the course (l48) example course-v1:openfun+mathematics101+session01.
L48 and L71 are different data, first one is context_id = lti_request.get_param("context_id"), and second is self.kwargs["uuid"]

Reply:
as discussed IRL I understood that lti_id is in one cas the LTI id for the context and in the other case the LTI id for the forum object.